webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -384,6 +384,15 @@ describe('ContributionForm.vue', () => {
           await wrapper.find('form').trigger('submit')
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
+
+        it('supports deleting a teaser image', async () => {
+          expectedParams.variables.image = null
+          propsData.contribution.image = '/uploads/someimage.png'
+          wrapper = Wrapper()
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:39:59Z" title="Thursday, January 16th 2020, 3:39:59 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -235,6 +235,12 @@ describe('ContributionForm.vue', () => {
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
 
+        it('deleted a teaser image', async () => {
+          expectedParams.variables.imageUpload = imageUpload
+          wrapper.find(TeaserImage).vm.$emit('addTeaserImage', imageUpload)
+          wrapper.find(TeaserImage).vm.$emit('deleteImage', null)
+        })
```


this isn't testing anything...
there must be an expectation, what happens when `deleteImage` is emitted?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T16:25:17Z" title="Tuesday, February 18th 2020, 5:25:17 pm +01:00">Feb 18, 2020</time>_

---


What is `expectedParams` here?